src/commands.ts

```diff
@@ -37,119 +36,35 @@ const commands: Record<string, Command> = {
         text: 'Shows help'
     },
     track: {
-        fn: async (msg, rawArgs) => {
-            const guildId: Snowflake | null = msg.guildId;
-            if (!guildId) {
-                await msg.reply('This command can only be used in a guild text channel!');
-                return;
-            }
-
-            const rsn = rawArgs && rawArgs.toLowerCase();
-            if (!rsn || !rsn.trim()) {
-                await msg.channel.send('Invalid username');
-                return;
-            }
-            if (state.isTrackingPlayer(guildId, rsn)) {
-                await msg.channel.send('That player is already being tracked');
-            } else {
-                await insertTrackedPlayer(guildId, rsn);
-                state.addTrackedPlayer(guildId, rsn);
-                await updatePlayer(rsn);
-                await msg.channel.send(`Now tracking player **${rsn}**`);
-            }
+        fn: async (msg) => {
+            await msg.channel.send('Use the **/track** command');
         },
         text: 'Tracks a player and gives updates when they level up',
         failIfDisabled: true
```

Comment:
Something that's missed by your PR: this `failIfDisabled` property refuses to perform a command if `state.isDisabled()` is true (which happens when the hiscores API changes)